src/Serilog.Extensions.Logging.File/Serilog.Extensions.Logging.File.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>Add file logging to ASP.NET Core apps with one line of code.</Description>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>3.1.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Can we also add `net5` here, to avoid pulling in the .NET Standard libraries? (Minimizes the dependency graph on new .NET SKUs)

Reply:
Perhaps, but INHO this should be done synchronously with the `Serilog.Extensions.Logging` package:

https://github.com/serilog/serilog-extensions-logging/blob/dev/src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj#L7

Reply:
Waiting to add this upstream just creates more delay/churn, though

Reply:
Done. I've added netcoreapp3.1 (supported till December, 2022), net5.0 (May, 2022) & net6.0 (November, 2024).

And have splitted corresponding package dependencies. The problem is that there are no recent dotnet SDKs on AppVeyor, so CI/CD failed.